src/settings.py

```diff
@@ -135,6 +135,7 @@ class GlobalSettings:
     grammar_search_score_function = "hff_lookahead_depth0"
     grammar_search_heuristic_based_weight = 10.
     grammar_search_heuristic_based_max_demos = 5
+    grammar_search_heuristic_based_max_nondemos = 50
```

Comment:
is there no hope of it ever running with the full 500, even in cover?

Reply:
we can make it env-dependent later if you want, I imagine 500 would be fine for cover, but 50 is also seemingly fine for cover